lib/random.gd

```diff
@@ -211,22 +225,30 @@ fi;
 ##  <Oper Name="RandomSource" Arg='cat[, seed]'/>
 ##
 ##  <Description>
-##  This operation is used to create new random sources. The first argument 
-##  <A>cat</A> is the category describing the type of the random generator, 
-##  an optional <A>seed</A> which can be an integer or a type specific data 
-##  structure can be given to specify the initial state.
+##  This operation is used to create new random sources. The first argument
+##  <A>cat</A> is the category describing the type of the random generator,
+##  for example one of the categories listed in
+##  Section <Ref Subsect="Kinds of Random Sources"/>.
+##  <P/>
+##  An optional <A>seed</A> can be given to specify the initial state.
+##  It can be an integer or a type specific data structure.
```

Comment:
Perhaps here we should just refer to the documentation of `Reset`?
```suggestion
##  For details, see  <Ref Oper="State"/> and  <Ref Oper="Reset"/>
```

Reply:
O.k.